tests/admin_views/tests.py

```diff
@@ -725,6 +725,15 @@ def test_disallowed_to_field(self):
         response = self.client.get(reverse('admin:admin_views_referencedbyinline_changelist'), {TO_FIELD_VAR: 'name'})
         self.assertEqual(response.status_code, 200)
 
+        # #25622 - Specifying a field of model that is only referred by a
```

Comment:
of a model
only referred to
and don't think we need "and not result in an AttributeError." -- how something crashed at one time isn't important for future reference.